app/queries/servers/system.ts

```diff
@@ -192,14 +192,14 @@ export const prepareCommonSystemValues = (
     }
 };
 
-export const setCurrentChannelId = async (operator: ServerDataOperator, channelId: string) => {
+export const setCurrentChannelId = async (operator: ServerDataOperator, channelId: string, prepareRecordsOnly = false) => {
```

Comment:
not sure why this change, in case you need to just prepare the models, you can just use `prepareCommonSystemValues` instead of calling `setCurrentChannelId`